src/Http/Controllers/HealthController.php

```diff
@@ -20,6 +20,10 @@ public function __construct(HealthCheckWrapper $healthWrapper)
         $this->HealthWrapper = $healthWrapper;
```

Comment:
Better structure the config, don't parse a string. With a good structure you can have multiple check for every linked system (2 database, 2 redis, more bucket s3 etc...)

the structure can be:

checks => [
   db => [

        [host => x.x.x.x, username => xxxx, password => xxxx]
   ],
   redis =>  null // if no redis is used
   etc...
]

Reply:
Another improve: you can create different classes to check every technology in different way. Every class has the same interfaces (check) but each one with own implementation. In this way your code is more "splitted"